aemdesign-aem-services/src/main/java/design/aem/utils/components/ComponentDetailsUtil.java

```diff
@@ -409,6 +409,9 @@ public static ComponentProperties getPageInfo(Map<String, Object> pageContext, P
                 componentProperties.putAll(getAssetInfo(resourceResolver,
                         getPageImgReferencePath(page),
                         FIELD_PAGE_IMAGE));
+                if(page.getProperties().get("mmalFragmentPath", String.class) != null)    {
```

Comment:
I don't think it can be approved as it contains project specific data

Reply:
I had talked with @cshawaus on this. And he told we can add few things if required. Applying this change in MMAL will take lot of time.

Reply:
Project specific changes can never be merged back into the core repo. A better solution would be to use a configurable OSGi service that you can specify the property name in.

Reply:
@saketkrnph what are you tryin to achieve with this? Would be good to describe what you are trying to do.

Right now you can just do this on specific components you need by using `ready` method. 

Also have you seen the `pageMetadataTab` tab on the Generic Detials component you can use that for any generic page level metadata and it would not require adding anymore properties to Page Properties dialog.